pom.xml

```diff
@@ -744,13 +744,11 @@
         <groupId>org.apache.logging.log4j</groupId>
         <artifactId>log4j-api</artifactId>
         <version>${log4j.version}</version>
-        <scope>${hadoop.deps.scope}</scope>
       </dependency>
       <dependency>
         <groupId>org.apache.logging.log4j</groupId>
         <artifactId>log4j-core</artifactId>
         <version>${log4j.version}</version>
-        <scope>${hadoop.deps.scope}</scope>
       </dependency>
       <dependency>
         <!-- API bridge between log4j 1 and 2 -->
```

Comment:
Then I think `log4j-slf4j-impl` and `log4j-1.2-api` are also needed?

Reply:
Yes. log4j-slf4j-impl is also needed:
https://github.com/apache/spark/blob/1431a4aa74cf69b3ee607e313dece6fed8390de6/core/src/main/scala/org/apache/spark/internal/Logging.scala#L228-L234

Reply:
If we include `lof4j-slf4j-impl` will we then not end up with multiple slf4j bindings on the classpath since hadoop already includes `slf4j-log4j12`?

Reply:
Thank you @eejbyfeldt . Removed it.